src/lib/autoinstall/clients/inst_autoinit.rb

```diff
@@ -150,8 +150,7 @@ def run
       # an ask-list is declared redoing the import and write of the pre-scripts as
       # many times as needed.
       def autoinit_scripts
-        # Pre-Scripts
-        Yast::AutoinstScripts.Import(Yast::Profile.current["scripts"] || {})
+        # pre-scripts should be already imported
```

Comment:
please write where it should be already imported. Ideally by what.

Reply:
maybe even add check for it ( it should be easy check like if profile scripts is non-empty, then autoinst scripts should be also non-empty )

Reply:
maybe we could even left the import here to prevent future errors because of moved code